app/assets/javascripts/application.js.erb

```diff
@@ -35,6 +35,7 @@
 //= require lib/jquery.colorcontrast
 //= require lib/jquery.trap
 //= require lib/jquery.is_visible_within
+//= require lib/jquery.fullscreen
```

Comment:
Instead requiring it through sprockets, install the package in the frontend folder witwh `npm install --save <package` and `require('package');` it in `frontend/app/vendors.js`